modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/ParentOrganization.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayForm, {ClayInput} from '@clayui/form';
+import {openSelectionModal, sub} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+export default function ParentOrganization({
+	label: initialLabel,
+	parentOrganizationId: initialParentOrganizationId,
+	portletNamespace,
+	selectOrganizationRenderURL,
+}) {
+	const [values, setValues] = useState({
```

Comment:
You can destructure the values here:

```suggestion
	const [{label, parentOrganizationId}, setValues] = useState({
```